app/src/main/AndroidManifest.xml

```diff
@@ -16,8 +16,9 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-        <activity android:name=".MainActivity"></activity>
-        <activity android:name=".UpdatePasswordActivity"></activity>
```

Comment:
mmm.... Please do not change things here... when code is injected, you should treat that code as if it was reviewed by US president, so never changing what is in project